index.js

```diff
@@ -1,24 +1,21 @@
 var sep = require('path').sep
-var last = require('lodash').last
 var toString = require('mdast-util-to-string')
 
 module.exports = checkTitle
 
-function checkTitle (remark, opts) {
+function checkTitle (opts) {
+  var title = (opts || {}).title
   return function checkTitleTransformer (root, file) {
+    var filePath = (file.dirname === '.' ? file.cwd : file.dirname) || title || process.cwd();
+    var dirnames = filePath.split(sep)
+    var folder = dirnames[dirnames.length - 1].toLowerCase()
```

Comment:
I agree. Just looked at the tests, and it doesn't actually check that the opt works - the files tested for equality are the same. Should be different. Can you fix that? Otherwise, this looks good.